internal/backends/winit/renderer/sw.rs

```diff
@@ -51,16 +50,15 @@ impl<const BUFFER_COUNT: usize> super::WinitCompatibleRenderer for SoftwareRende
                 type TargetPixel = i_slint_core::swrenderer::Rgb565Pixel;
                 fn process_line(
                     &mut self,
-                    line: PhysicalLength,
-                    range: core::ops::Range<PhysicalLength>,
+                    line: usize,
+                    range: core::ops::Range<usize>,
                     render_fn: impl FnOnce(&mut [Self::TargetPixel]),
                 ) {
-                    let len = (range.end.get() - range.start.get()) as usize;
-                    let line_begin = line.get() as usize * self.line.len();
+                    let len = range.end - range.start;
```

Comment:
```suggestion
                    let len = range.len();
```

And maybe even no need to have a variable for that then.

Reply:
Good idea